CHARTER.md

```diff
@@ -1,51 +1,61 @@
+### Chair Participation
+
+Chairs are expected to remain actively involved in the community. A chair who does not participate in ART voting *without reason and notice* three times in a row forfeits the position and becomes a regular ART member automatically.
+
+
+## Proposals
+
+Proposals may only be made asychronously by submitting a pull request to this repository with a new file in the [`art-decisions/proposals`](art-decisions/proposals) directory. The provided [template file](art-decisions/proposals/template.md) may be used as a guideline for new proposals. Proposals may only be submitted by current ART members. If you would like to encourage discussion around a proposal before any voting starts, please submit a draft pull request instead, and mark it ready for review once you are comfortable that you will not make any changes to your proposal. Open pull requests can be viewed at the [ART repository pull requests](https://github.com/arcalot/arcalot-round-table/pulls) page.
 
 ## Voting
 
-Each member of the Arcalot Round Table is eligible for voting if they have been a member of the ART for at least 30 days. For any decision that impacts the whole project or processes, a vote is started where a 66% majority with a quorum of 66% of all ART members is needed for the proposal to pass.
+Each member of the ART is eligible to vote if they have been a member for at least 30 days. For any proposal that impacts the whole project or processes, **approval requires a minimum of four votes with 75% approval among those voting**.
 
-If the majority cannot be reached, the proposal is considered rejected and can be brought to the ART again 2 months after the rejection at the earliest. If the same decision has been rejected twice, it cannot be brought to the ART again.
+Any vote must be open for at least 7 calendar days to account for different time zones and other obligations and must be open no longer than 30 days. The voting period must be stated in the proposal in calendar days. The voting period begins on the day the pull request is opened for review and ends at midnight [AoE](https://en.wikipedia.org/wiki/Anywhere_on_Earth) on the last day of the voting period. Proposals can be extended for no longer than 30 days total. Extensions can be made before the end of the current voting period by the original submitter, their direct representative, or one of the ART chairs by updating the pull request with a note stating the number of calendar days for the extension. **If approval is not reached within the voting period deadline, the proposal is rejected.**
 
```

Comment:
Using UTC like this means that a proposal could be opened from the US West coast with a 7 day voting period, and that period would end for that same time zone at 5:00 PM on the 7th day.

The end time certainly shouldn't be tied to the timezone of the submitter since that is too complicated to administer. And truly we are talking about dates and not times; midnight is simply a shortcut reference to be clear about the end of day. But we could simply say end-of-day AoE or otheriwse UTC-12/+12 (or [UTC+14](https://en.wikipedia.org/wiki/UTC%2B14:00))

Reply:
Operationally, the question we need anyone to be able to answer easily is "how long do I have to finish reviewing?" It's not "the end of July 17"; or at least, not in any conventional and obvious sense. It's "the end of July 17 in some obscure Pacific island chain". If I'm in UTC that deadline is meaningless and useless: I've actually got until noon the following calendar day.

So far as I can see, this is just locking our reviews to a specific time zone, but being deliberately difficult by choosing an obscure time zone where hardly anyone lives instead of something universally understood like UTC. I see no possible actual benefit here, just more words and complication.

Reply:
I concede the point.